index-fr.html

```diff
@@ -164,8 +163,8 @@ <h2>Pourquoi les masques nous protègent doublement</h2>
   <div class="container">
 
     <p>
-      Quand on y pense, c'est surprenant d'un masque efficace à 50% puisse réduire le risque d'infection de 75%.
-      C'est possible parce que lorsque les deux personnes portent des masques, la probabilité d'infection est divisée deux fois.
+      Quand on y pense, c'est surprenant qu'un masque efficace à 50% puisse réduire le risque d'infection de 75%.
+      C'est possible parce que lorsque les deux personnes portent des masques, la probabilité d'infection est divisée par deux.
```

Comment:
Here we are trying to say that the infection probability was reduced twice (once for the first person, then another time for the other person). It's different from saying that the probability was divided by two.